tests/local/e2e_test.go

```diff
@@ -34,83 +34,83 @@ func TestE2E(t *testing.T) {
 
 // Define the Teleporter before and after suite functions.
 var _ = ginkgo.BeforeSuite(func() {
-	localNetworkInstance = newLocalNetwork(warpGenesisFile)
+	LocalNetworkInstance = NewLocalNetwork(warpGenesisFile)
 	// Generate the Teleporter deployment values
 	teleporterDeployerTransaction, teleporterDeployerAddress, teleporterContractAddress, err :=
 		deploymentUtils.ConstructKeylessTransaction(teleporterByteCodeFile, false)
 	Expect(err).Should(BeNil())
 
-	_, fundedKey := localNetworkInstance.GetFundedAccountInfo()
-	localNetworkInstance.deployTeleporterContracts(
+	_, fundedKey := LocalNetworkInstance.GetFundedAccountInfo()
+	LocalNetworkInstance.DeployTeleporterContracts(
 		teleporterDeployerTransaction,
 		teleporterDeployerAddress,
 		teleporterContractAddress,
 		fundedKey,
 	)
-	localNetworkInstance.deployTeleporterRegistryContracts(teleporterContractAddress, fundedKey)
+	LocalNetworkInstance.DeployTeleporterRegistryContracts(teleporterContractAddress, fundedKey)
 	log.Info("Set up ginkgo before suite")
 })
 
 var _ = ginkgo.AfterSuite(func() {
-	localNetworkInstance.tearDownNetwork()
+	LocalNetworkInstance.TearDownNetwork()
 })
 
 var _ = ginkgo.Describe("[Teleporter integration tests]", func() {
 	// Cross-chain application tests
 	ginkgo.It("Send native tokens from subnet A to B and back", func() {
-		flows.NativeTokenBridge(localNetworkInstance)
+		flows.NativeTokenBridge(LocalNetworkInstance)
 	})
 	ginkgo.It("Send ERC20 tokens from subnet A to Native tokens on subnet B and back", func() {
-		flows.ERC20ToNativeTokenBridge(localNetworkInstance)
+		flows.ERC20ToNativeTokenBridge(LocalNetworkInstance)
 	})
```

Comment:
These two test cases currently fail when using the C-Chain on one end. I need to investigate this further.

Reply:
Fixed in https://github.com/ava-labs/teleporter/pull/207